src/autocomplete/dataset.js

```diff
@@ -93,7 +93,7 @@ _.mixin(Dataset.prototype, EventEmitter, {
     var renderArgs = [].slice.call(arguments, 2);
     this.$el.empty();
 
-    hasSuggestions = suggestions && suggestions.length;
+    hasSuggestions = suggestions && suggestions.length > 0;
```

Comment:
I am not sure to understand how this fixes a bug, can you give more precisions on:
- what are the types suggestions could be? If it's always an array this test should be sufficient, but maybe there are cases where it's not an array?

If so, we should do an isArray check somewhere